manifests/pipecd/values.yaml

```diff
@@ -109,10 +109,48 @@ cors:
 quickstart: false
 
 monitoring:
+  # If true, prometheus and grafana sub-charts will be installed
   enabled: false
 
 # All directives inside this section will be directly sent to the prometheus chart.
 # Head to the below link to see all available values.
 # https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/values.yaml
-# TODO: Prepare the default prometheus.yml
-prometheus: {}
+prometheus:
+  server:
+    fullnameOverride: pipecd-prometheus-server
+  pushgateway:
+    enabled: false
+  nodeExporter:
+    enabled: false
+  kubeStateMetrics:
+    enabled: false
+  configmapReload:
+    prometheus:
+      enabled: true
+  serverFiles:
+    prometheus.yml:
+      rule_files:
+        - /etc/config/recording_rules.yml
+        - /etc/config/alerting_rules.yml
+
+      scrape_configs:
+        - job_name: prometheus
+          static_configs:
+            - targets:
+                - localhost:9090
+
+        - job_name: pipecd-gateway
```

Comment:
Can you add jobs for `server`  and `ops` too?
Both of them are exposing the metrics at the `admin` port and `/metrics` path.

Reply:
Got it!